src/lib/pages/backoffice/Document/DocumentDetails/DocumentContent.js

```diff
@@ -2,8 +2,10 @@ import { DocumentEItems } from '@pages/backoffice/Document/DocumentDetails/Docum
 import { DocumentItems } from '@pages/backoffice/Document/DocumentDetails/DocumentItems';
 import { DocumentPendingLoans } from '@pages/backoffice/Document/DocumentDetails/DocumentPendingLoans';
 import { DocumentStats } from '@pages/backoffice/Document/DocumentDetails/DocumentStats';
-import DocumentSeries from '@pages/backoffice/Document/DocumentDetails/DocumentRelations/DocumentSeries';
-import DocumentSiblings from '@pages/backoffice/Document/DocumentDetails/DocumentRelations/DocumentSiblings';
+import {
+  DocumentSeries,
+  DocumentSiblings,
+} from '@pages/backoffice/Document/DocumentDetails/DocumentRelations';
```

Comment:
just for info, we have tried to remove the `index.js`, whenever possible...

Reply:
These two are the only case being exported from the same `index.js`, for the rest of the *Relation* components with their modals they are exported from their index in order to connect them with the reducer. We can even deprecate completely the `index.js` and connect them in their source file, which would also prevent us from importing from the wrong place, i.e this case where the components were imported explicitly though, not connected to their reducers.